site/docs/numbers/porting/csrLookupTool.mdx

```diff
@@ -32,47 +32,18 @@ This walks through how to programmatically lookup Customer Service Records (CSRs
 
 ## Polling vs. Webhooks
 
-CSR Orders in the Bandwidth Dashboard are asynchronous when creating an "order". The orders are then processed and the order status is updated asynchronously.  Bandwidth recommends configuring your account with a subscription instead of polling the order resource for `<OrderStatus>`.
+CSR Orders in the Bandwidth Dashboard are asynchronous when creating an "order". The orders are then processed and the order status is updated asynchronously. Bandwidth recommends configuring your account with a subscription instead of polling the order resource for `<OrderStatus>`.
 
 Order processing times can vary and are not guaranteed, so bandwidth does __not__ recommend setting a timeout on waiting for an order to show either `COMPLETE` or `FAILED` status, but instead relying on a webhook from a CSR Orders subscription.
```

Comment:
Suggest changing this to just "Order processing times can vary and are not guaranteed." and removing the rest.

Reply:
Having Riley's comment as well I put it as:

```
CSR orders are processed and updated asynchronously. Processing times can vary. For status updates, it is recommended to use subscriptions versus polling the order resource for `<OrderStatus>`.
```

sounds ok?

Reply:
Sounds good